tensorvision/utils.py

```diff
@@ -54,12 +58,14 @@ def set_dirs(hypes, hypes_fname):
         else:
             runs_dir = os.path.join(base_path, 'RUNS')
 
-        if not FLAGS.save:
+        if not FLAGS.save and FLAGS.name is None:
```

Comment:
When FLAGS.save is False, shouldn't this be executed no matter what FLAGS.name is? (hence "or", not "and"?)